playground/fs-serve/root/vite.config-base.js

```diff
@@ -0,0 +1,36 @@
+import path from 'node:path'
+import { defineConfig } from 'vite'
+
+const BASE = '/base'
```

Comment:
We should better use `/base/` here. People check the playgrounds to try out features, so it is good to align with the way we have it in the config docs.

Reply:
got it